pygdf/dataframe.py

```diff
@@ -430,7 +430,7 @@ def _concat(cls, objs, ignore_index=False):
             what = set(o.columns for o in objs)
             raise ValueError('columns mismatch: {}'.format(what))
         # Filter out inputs that have 0 length
```

Comment:
Need to remove this comment.

Reply:
you should yell at the guy that created this PR :P